docker/jenkins/run_functional_tests_sauce.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Starts Selenium Hub and NUMBER_OF_NODES (default 5) firefox nodes.
+# Waits until at least one node is ready and then runs sanity tests
+# only against a local bedrock instance started for this job.
+#
+set -xe
+
+cp docker/dockerfiles/bedrock_functional_tests Dockerfile
+docker build -t bedrock_functional_tests:${GIT_COMMIT} --pull=true .
+docker run -v `pwd`/results:/app/results \
+-e BASE_URL=${BASE_URL} \
```

Comment:
Indent a couple of columns in to make it clear that this a continuation of the line above.